README.md

```diff
@@ -571,6 +571,7 @@ Not all options are turned on by default. Refer to `TR1X_ConfigTool.exe` for det
 - added contextual arrows to menu options
 - added support for animated room sprites, which also restores intended behavior in, for example, The Cistern room 0
 - added skybox support, with a default option provided for Lost Valley, Colosseum and Obelisk of Khamoon; custom level builders can use object slot `184`
+- added reflections of Midas Hand death animation and savegame crystals
```

Comment:
I think `added reflections to Midas Hand death animation and savegame crystals` makes more sense